src/components/specific/files/files-manager/files-manager-breadcrumb/FilesManagerBreadcrumb.vue

```diff
@@ -15,6 +17,7 @@
       :text="file.name"
       :maxLength="24"
       tooltipColor="tertiary-lightest"
+      :style="{ marginLeft: path.length ? '15px' : '5px' }"
```

Comment:
this property permit to the folder to be aligned with "Nom", when the user is at the root folder. Otherwise, it take the default margin -> 15px

Reply:
Nice ! :)
It seems to be a good idea to align folders with the first column header in any case (whether we are in the root folder or not).